src/MeetingsSDKAdapter.js

```diff
@@ -222,7 +348,16 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
         map(() => this.meetings[ID])
       );
 
-      const getMeetingWithEvents$ = concat(getMeeting$, meetingWithReadyEvent$).pipe(
+      // Listen to mute event to return the meeting object
+      const meetingWithLocalMuteEvents$ = fromEvent(sdkMeeting, EVENT_MEDIA_LOCAL_MUTE).pipe(
+        map(() => this.meetings[ID])
+      );
+
+      // Merge all event observables to update the existing meeting object simultaneously
+      const meetingsWithEvents$ = merge(meetingWithReadyEvent$, meetingWithLocalMuteEvents$);
+
+      const getMeetingWithEvents$ = getMeeting$.pipe(
```

Comment:
I don't think this should be a `pipe`, this should still be a `concat` but with `meetingsWithEvents$`

Reply:
addressed